python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -779,7 +1040,13 @@ def assemble(self, skyInfo, tempExpRefList, imageScalerList, weightList, bgModel
         return coaddExp
 
     def buildDifferenceImage(self, skyInfo, tempExpRefList, imageScalerList, weightList, bgModelList):
-        """Return an exposure that contains the difference between and unclipped and clipped coadd
+        """!
+        \brief Return an exposure that contains the difference between and unclipped and clipped coadds.
+
+        Generate a difference image between clipped and unclipped coadds.
+        Compute the difference image by subtracting an outlier-clipped coadd from an outlier-unclipped coadd.
+        Set the PSF of the difference image to that of the outlier unclipped coadd and return the difference
```

Comment:
I don't think you need to mention that the fact that the PSF is set.  This is done as an implementation detail so that we can run the detection algorithm on an image.